intelmq/lib/bot.py

```diff
@@ -394,5 +399,56 @@ def __load_harmonization_configuration(self):
                             "Key %s is not valid." % _key)
 
 
+class ParserBot(Bot):
+
+    def parse_csv(self, report):
```

Comment:
where does this get used? I like it but ... should we not have a subclass of ParseBot called CSVParseBot for that?